capture/fswebcam.py

```diff
@@ -0,0 +1,90 @@
+import subprocess
+
+import capture
+from utils.log import debug
+
+
+class Fswebcam(capture.ImgCapture):
```

Comment:
Well, not a big fan: I prefer to use the package namespaces for this: these classes will be called through `capture.Fswebcam`, `capture.Mock`, etc.